target-gen/src/generate.rs

```diff
@@ -179,10 +179,13 @@ fn create_core(processor: &Processor) -> Result<ProbeCore> {
                 debug_base: None,
                 cti_base: None,
             }),
-            Architecture::Riscv => {
-                CoreAccessOptions::Riscv(RiscvCoreAccessOptions { hart_id: None })
+            Architecture::Riscv => CoreAccessOptions::Riscv(RiscvCoreAccessOptions {
```

Comment:
Is this not contained in the pdsc?

Reply:
I don't know how things relate to each other here. I don't know if there are risc-v targets with CMSIS packs, I didn't do any research here. We'll find out when we run into one :)

Reply:
Ah true, we only have SVDs, not Packs.